test/sourceMap-option.test.js

```diff
@@ -201,20 +205,22 @@ describe('sourceMap', () => {
     expect(getWarnings(stats)).toMatchSnapshot('warnings');
   });
 
-  it('should match snapshot when the "devtool" option has the "sourcemap" value', async () => {
-    const compiler = getCompiler({
-      entry: `${__dirname}/fixtures/entry.js`,
-      devtool: 'sourcemap',
-    });
+  if (getCompiler.isWebpack4()) {
```

Comment:
Why only webpack@4?

Reply:
Cause devtool: sourcemap is not valid value anymore

Reply:
Let's fix it in test - rename `sourcemap` to  `source-map`